pkg/controller/clone-controller.go

```diff
@@ -468,7 +474,8 @@ func createCloneSourcePodName(targetPvc *corev1.PersistentVolumeClaim) string {
 
 // MakeCloneSourcePodSpec creates and returns the clone source pod spec based on the target pvc.
 func MakeCloneSourcePodSpec(image, pullPolicy, sourcePvcName, sourcePvcNamespace, ownerRefAnno string,
-	clientKey, clientCert, serverCACert []byte, targetPvc *corev1.PersistentVolumeClaim, resourceRequirements *corev1.ResourceRequirements) *corev1.Pod {
+	clientKey, clientCert, serverCACert []byte, targetPvc *corev1.PersistentVolumeClaim, resourceRequirements *corev1.ResourceRequirements,
+	workloadNodePlacement cdiv1.NodePlacement) *corev1.Pod {
```

Comment:
Any particular reason you are dereferencing the workloadNodePlacement above? Instead of passing a pointer as a parameter you are now passing the entire struct. I think you can have workloadNodePlacement *cdiv1.NodePlacement without any code changes in the actual function.

Reply:
No reason for it. changed.